photospp-toolfile.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external photospp-toolfile 3.56
+### RPM external photospp-toolfile 2.0
```

Comment:
@mkirsano the version number is different in spec file than tool-file , shouldn't it be the same?